networking_ccloud/tests/common/config_fixtures.py

```diff
@@ -128,10 +128,30 @@ def make_azs_from_switchgroups(switchgroups):
     return make_azs(az for az in set(sg.availability_zone for sg in switchgroups))
 
 
+def make_vrfs(names):
+    vrfs = []
+    for i, name in enumerate(names):
+        vrfs.append(config_driver.VRF(name=name, number=i + 1))
+    return vrfs
+
+
+def make_vrfs_from_infra_nets(infra_nets):
+    if infra_nets is None:
+        return []
+
+    return make_vrfs(x.vrf for x in set(infra_nets) if x.vrf)
+
+
 # whole config
 def make_config(switchgroups=None, hostgroups=None, global_config=None):
     if not global_config:
-        global_config = make_global_config(availability_zones=make_azs_from_switchgroups(switchgroups))
+        infra_nets = list()
+        if hostgroups:
+            for hostgroup in hostgroups:
+                if hostgroup.infra_networks:
+                    infra_nets.extend(hostgroup.infra_networks)
```

Comment:
Could also make a `make_vrfs_from_hostgroups()` instead of `from_infra_nets`. Maybe that's more our use case?